src/main/java/ratpack/zipkin/internal/ServerHttpAdapter.java

```diff
@@ -1,13 +1,24 @@
 package ratpack.zipkin.internal;
 
+import com.google.common.net.HttpHeaders;
 import ratpack.zipkin.ServerRequest;
 import ratpack.zipkin.ServerResponse;
+import zipkin2.Endpoint;
 
 /**
  * This class is responsible for adapting Ratpack-specific request and responses
  * to something that brave.http.HttpServerParser can use to create the Span.
  */
 final class ServerHttpAdapter extends brave.http.HttpServerAdapter<ServerRequest, ServerResponse> {
+  @Override
+  public boolean parseClientAddress(final ServerRequest serverRequest, final Endpoint.Builder
```

Comment:
killme-nit: wonder if removing final qualifiers it will not wrap :P

Reply:
lol, I can just move the param to a new line and line them up...